src/hooks/useCareApps.tsx

```
@@ -37,16 +39,19 @@ const withSuspense = (Component: React.ComponentType<EncounterTabProps>) => {
-  return careApps.reduce((acc, app) => {
-    const appTabs = Object.entries(app.encounterTabs ?? {}).reduce(
-      (acc, [key, Component]) => {
-        return { ...acc, [key]: withSuspense(Component) };
-      },
-      {},
-    );
+  return careApps.reduce<Record<string, React.FC<PluginEncounterTabProps>>>(
+    (acc, app) => {
+      const appTabs = Object.entries(app.encounterTabs ?? {}).reduce(
+        (acc, [key, Component]) => {
+          return { ...acc, [key]: withSuspense(Component) };
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)